amethyst_assets/src/loader.rs

```diff
@@ -157,7 +157,8 @@ impl Loader {
         handle
     }
 
-    fn source(&self, source: &str) -> Arc<Source> {
+    // TODO: consider exposing
```

Comment:
Can you elaborate on why we chose not to expose this and why that reasoning might change in the future?

Reply:
I did expose it now, there's no reason to not do this.